config/config/sc-config.yaml

```diff
@@ -472,7 +468,6 @@ prometheus:
 
 dex:
   replicaCount: 2
-  subdomain: dex
```

Comment:
can you add some comments saying that the `subdomain` key can be found in the common config?

Reply:
done in https://github.com/elastisys/compliantkubernetes-apps/pull/1700/commits/d0df25d4e6f47a209cae55b835f7f2e4d13eba44